views/default/default/layout.ejs

```diff
@@ -115,11 +115,11 @@
 <% if (typeof title!== 'undefined') { %>
 <% } %>
  
-<% if (sails.config.brandingAware(branding).systemMessage.enabled == true) { %>
+<% if (sails.config.brandingAware(BrandingService.getBrandFromReq(req)).systemMessage.enabled === true) { %>
```

Comment:
This change from `branding` to `BrandingService.getBrandFromReq(req)` was the quickest way to allow the error page to load without `branding` set as a sails local.
The branding does seem to be available from one of the sources checked by the method.